.mk/development.mk

```diff
@@ -69,3 +69,35 @@ undeploy-infra: undeploy-loki undeploy-grafana uninstall
 
 .PHONY: undeploy-all
 undeploy-all: undeploy-infra undeploy-sample-cr undeploy-sample-workload
+
+.PHONY: generate-flp-configuration
```

Comment:
the `generate-flp-configuration` target doesn't belong to `development` it needs to be part of either the main Makefile or a new file under .mk folder called something like flowlogspipline.mk - in any event, this is not part of the development process.

Reply:
moved it to flp.mk